src/util/color/rgbcolor.h

```diff
@@ -0,0 +1,153 @@
+#pragma once
+
+#include <QColor>
+#include <QVariant>
+#include <QtDebug>
+#include <QtGlobal>
+
+#include "util/assert.h"
+#include "util/optional.h"
+
+namespace mixxx {
```

Comment:
@uklotzde Is it ok if I move this class into the Color namespace in my cue color branch?

Reply:
Please coordinate this with @Holzhaus to avoid merge conflicts. Better do it in a separate PR that is expected to be merged quickly.`

Reply:
Please also note that namespace names should be lowercase:

https://google.github.io/styleguide/cppguide.html#Namespace_Names

The `Color` namespace violates this rule.

Reply:
@ferranpujolcamins Do we actually need a separate `color` namespace within `mixxx` for just a bunch of classes, all with _color_ in their class name?

Reply:
The color namespace was there to group some free functions. We can reuse it to include your class. But it doesn't really matter to me. Just let me know what you think it's best.

Reply:
With argument dependent lookup and function overloading available in C++ there is no need to hide free functions in artificial namespaces. Or do I miss a use case?

We are free to chose whatever works best for us. However keep an eye on discoverability and developer experience in general.